packages/sdk/src/adapters/multipeer/rules.ts

```diff
@@ -4,6 +4,7 @@
  */
 
 import deepmerge from 'deepmerge';
+import semver from 'semver';
```

Comment:
I don't see semver being used any longer here after the backwards compat removal.  Safe to remove this?